_episodes/01-intro.md

```diff
@@ -32,17 +32,10 @@ including through a keyboard and mouse, touch screen interfaces, or using speech
 While touch and voice interfaces are becoming more commonplace, most interaction is still
 done using traditional screens, mice, touchpads and keyboards.
 
-We are all familiar with **graphical user interfaces** (GUI): windows, icons and pointers.
-They are easy to learn and fantastic for simple tasks where a vocabulary consisting of
-"click" translates easily into "do the thing I want". But this magic relies on 
-wanting a simple set of things, and having programs that can do exactly those things.
+The **graphical user interface** (GUI) is the most widely used way to interact with personal computers. We give instructions (to run a program, to copy a file, to create a new folder/directory) with the convenience of a few mouse clicks. This way of interacting with a computer is intuitive and very easy to learn. But this way of giving instructions to a computer scales very poorly if we are to give large stream to instructions even if they are similar or identical. For example if we have to copy third line of each of a thousand text files stored in thousand different folders/directories and paste it into a single file line by line. Using the tradition GUI approach of clicks will take several hours to do this.  
```

Comment:
large stream to instructions > large stream *of* instructions
if we have to copy third line > if we have to copy *the* third line